rac-gcp-deploy/orb.yml

```diff
@@ -765,6 +765,57 @@ jobs:
             IMAGE_ROOT=<<parameters.registry-url>>/$<<parameters.gcp-project-id>>/$<<parameters.container-name>>
             docker push "${IMAGE_ROOT}"
 
+  deploy_to_kmi:
+    parameters:
+      workspace-dir:
+        description: "Path to restore/save the workspace"
+        type: string
+        default: ~/project
+      container-name:
+        description: "Name of environment variable storing the name of the container we are publishing"
+        type: env_var_name
+        default: CIRCLE_PROJECT_REPONAME
+      kmi-env:
+        description: "The environment of KMI to deploy to"
+        type: string
+        default: uat
+      git-username:
+        description: "Name of the environment variable storing the github user to use when pushing commits"
+        type: env_var_name
+        default: GIT_USERNAME
+      git_user_email:
```

Comment:
this uses `_` instead of `-` like the other parameters

Reply:
The other jobs in the orb also use `git_user_email` - I can't find anywhere in the ovotech organization that's explicitly setting it though, as it looks like it's only being set through the environment variable, so I've changed it in all the jobs.